--- repo_orig/android-ir-blaster-master/lib/main.dart	2026-08-31 23:22:51.000000000 +0000
+++ repo/android-ir-blaster-master/lib/main.dart	2026-09-05 21:25:14.312933372 +0000
@@ -9,6 +9,7 @@
 import 'package:irblaster_controller/state/app_theme.dart';
 import 'package:irblaster_controller/state/dynamic_color.dart';
 import 'package:irblaster_controller/state/haptics.dart';
+import 'package:irblaster_controller/state/network_bridge_prefs.dart';
 import 'package:irblaster_controller/state/orientation_pref.dart';
 import 'package:irblaster_controller/state/remote_display_prefs.dart';
 import 'package:irblaster_controller/state/startup_prefs.dart';
@@ -50,6 +51,7 @@
       RemoteDisplayController.instance.load(),
       StartupPrefsController.instance.load(),
       TransmitterPrefs.instance.load(),
+      NetworkBridgePrefs.instance.load(),
       // lazy import to avoid circulars; we refer by string to keep tool happy
     ]);
   } catch (e, st) {
